### DISCUSSION TITLE
Expose full PyVista capability for Fluent post processing

### DISCUSSION THREAD

**Discussion by mkvani** [ansys/pyfluent#334] [Ideas]
Hi,

As per my exploration of Fluent PyVista module (ansys.fluent.post.pyvista), post processing features are very limited. For example, 

1) Contour number formatting is not available (#298 )
2) View setting
3) Lighting

I assume we are writing a wrapper on PyVista for Fluent data post processing.
If that is true, then there will be several such features which will need to be developed. Ideally we would need to support all features available in Fluent post processing under this wrapper. Even after that, I assume, we will not be able to use core PyVista capability.

Instead of that, is it a good idea to have a functionality where we can write a Fluent cas/dat to the format which PyVista supports? If yes, it will be really helpful  to leverage PyVista capability. 

Best regards,
Manoj

**Comment by dnwillia-work** [comment]
Hi Manoj, we are writing a wrapper but are the missing features something that is available in PyVista and is not in the wrapper or is it not available in PyVista?

**Comment by dnwillia-work** [comment]
@mkvani Sorry, I should have said that if there is something else you need which is not available in the wrapper please start putting issues in the pyfluent-visualization repo.

**Reply by mkvani** [reply]
Thank you @dnwillia-work . Aseem updated in #298 that PyVista plotter has been exposed. I think that will be helpful.
